src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/ContainerDAO.scala

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import java.io.File
+
+import akka.actor.ActorSystem
+import com.google.api.client.util.store.FileDataStoreFactory
+import com.typesafe.config.{ConfigFactory, Config}
+
+/**
+ * Created by mbemis on 8/19/15.
+ */
+class ContainerDAO(methodRepoServer: String, executionServiceServer: String) {
```

Comment:
in ContainerDAO you already read values from conf, you might want to just read the values for methodRepoServer and executionServiceServer here instead of in Boot.scala:

```conf.getConfig("methodrepo").getString("server"), conf.getConfig("executionservice").getString("server")```